test/integration/scheduler/volume_binding_test.go

```diff
@@ -99,7 +99,7 @@ func TestVolumeBinding(t *testing.T) {
 		"VolumeScheduling":       true,
 		"PersistentLocalVolumes": true,
 	}
-	config := setupCluster(t, "volume-scheduling-", 2, features, 0, 0, false)
+	config := setupCluster(t, "volume-scheduling-", 2, features, 0, 0, true)
```

Comment:
if we're changing all these callsites anyway, can we just drop the equivalencecache arg and related code from setupCluster and initTestSchedulerWithOptions in this PR?

Reply:
As we chatted offline, since the equivalence cache is still an alpha feature that can be enabled, I would prefer leaving the param for now until we delete the feature.